docs/sources/flow/upgrade-guide.md

```diff
@@ -21,6 +21,12 @@ Grafana Agent Flow.
 
 ## v0.35
 
+### Breaking change: Jaeger remote sampling does not work
+
+Jaeger remote sampling used to be configured via the jaeger receiver. This receiver was updated to a new version, and remote sampling has stopped working.
+
+Jaeger remote sampling will be reintroduced in v35.3. It will be configured separately from the jaeger receiver.
```

Comment:
I don't know if we should introduce new functionality in a patch release; it goes against semver. We could say this is a known issue in v0.35 and to wait until v0.36 for a fix.

Reply:
If it is something that is currently broken then fixing it in a patch via new functionality feels reasonable. The other option that is most semver compatible is to roll back the otel upgrade. Given the effort there that is painful to do. @ptodev have we gotten any reports on this or did we find it? 

I guess fundamentally though we could roll out a v0.36 instead of a patch if we wanted to align with semver?

Reply:
Yeah I'm not going to push too hard on this, but it does feel bad to categorize it as a breaking change, since that feels more deliberate/planned, and this is just an artifact of upgrading.